cli/monitors/terminal_outputter.py

```diff
@@ -279,9 +279,9 @@ class StepListPrinter(MonitorListener):
     uses its own thread to allow the output of time clock counters for each step
     """
     # pylint: disable=C0103
-    OK = PP.green(PP.bold(u"\u2713"))
-    FAIL = PP.red(u"\u274C")
-    WAITING = PP.orange(u"\u23F3")
+    OK = PP.green(PP.bold(u"\u2713")) if check_terminal_utf8_support() else PP.green("OK")
+    FAIL = PP.red(u"\u274C") if check_terminal_utf8_support() else PP.red("Fail")
+    WAITING = PP.orange(u"\u23F3") if check_terminal_utf8_support() else PP.orange("Waiting")
```

Comment:
I guess it's not really a performance problem to call this function three times in a row, but since the output is guaranteed (?) not to change between calls, wouldn't it be cleaner to call it just once and save the output in a local variable?

Reply:
Yes, you're right, I'll change that.